test/test_multiprocessing.py

```diff
@@ -230,6 +239,8 @@ def test_receive():
             for _ in range(repeat):
                 test_fill()
                 test_receive()
+            if TEST_CUDA_IPC:
+                torch.cuda.ipc_collect()
```

Comment:
Should `leak_checker` automatically run `torch.cuda.ipc_collect()` before testing for leaks? That seems more robust.